examples/custom-congestion-controller/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "custom-congestion-controller"
+version = "0.1.0"
+authors = ["AWS s2n"]
+edition = "2021"
+
+[dependencies]
+s2n-quic = { version = "1", path = "../../quic/s2n-quic", features = ["unstable-congestion-controller"] }
```

Comment:
So that this example just works I think also include the .cargo/config.toml in the example. https://github.com/aws/s2n-quic/tree/main/examples/unreliable-datagram/.cargo

Reply:
Is that needed? We already have one at the root level that should enable support for unstable: https://github.com/aws/s2n-quic/blob/main/.cargo/config.toml

Reply:
I believe so since the example define their own workspace and so dont inherit from parent

```
[workspace]
members = ["."]
```

Reply:
The config is inherited from the parent workspace, but I've added it to the example code as well so it works if copy-pasted into a different workspace